prod/westeurope/internal/api/functions_public_r3/function_app_slot_staging/terragrunt.hcl

```diff
@@ -0,0 +1,95 @@
+dependency "function_app" {
```

Comment:
Wrong runtime version. Must be v3.

Reply:
@uolter Please also set autoswap

Reply:
> Wrong runtime version. Must be v3.

it's already referencing version 3:
```
dependency "function_app" {
  config_path = "../function_app"
}


.
├── function_app
│   └── terragrunt.hcl
├── function_app_slot_staging
│   └── terragrunt.hcl
└── subnet
    └── terragrunt.hcl

```

Reply:
> @uolter Please also set autoswap

autoswap is already set at line 62:

 auto_swap_slot_name = "production"

Reply:
1) the runtime version must be set on the slot
2) autoswap need a newer version of the module?